fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/VehicleFacade.java

```diff
@@ -588,5 +596,52 @@ public void moveToXY(String vehicleId, CartesianPoint cartesianPoint, double ang
         }
     }
 
+    /**
+     * Read all taxi relevant data for the given vehicle id.
+     */
+    public TaxiVehicleData getTaxiData(String vehicleId) throws InternalFederateException {
+        final int taxiState = Integer.parseInt(getParameter(vehicleId, "device.taxi.state"));
+        final int numPersonsServed = Integer.parseInt(getParameter(vehicleId, "device.taxi.customers"));
+        final String currentCustomers = getParameter(vehicleId, "device.taxi.currentCustomers");
+
+        return new TaxiVehicleData(
+                vehicleId,
+                TaxiVehicleData.TaxiState.of(taxiState),
+                getPersonCapacity(vehicleId),
+                bridge.getSimulationControl().getLastKnownVehicleData(vehicleId),
+                numPersonsServed,
+                Arrays.stream(StringUtils.split(currentCustomers, ',')).map(Bridge.PERSON_ID_TRANSFORMER::fromExternalId).toList()
+        );
+    }
+
+    private int getPersonCapacity(String vehicleId) throws InternalFederateException {
+        try {
+            return getPersonCapacity.execute(bridge, vehicleId);
+        } catch (IllegalArgumentException | CommandException e) {
+            throw new InternalFederateException("Could not request person capacity for vehicle " + vehicleId, e);
+        }
+    }
+
+    /**
+     * Dispatches taxi to pick up and drop off customers in a specific order.<br>
+     * Valid example for the reservation list:
+     * <ul>
+     *     <li>[a,b,c,d] - picks up and drops off in this order</li>
+     *     <li>[a, b, a, c, b, d, c, d] - picks first a and b, then drops off a,
```

Comment:
I don't really get this example.
This just means the order of pick-up is fixed but drop-offs can occur after picking up other customers?! How is this behavior determined, the dispatching algorithm or internally by SUMO?